lib/galaxy/webapps/galaxy/controllers/workflow.py

```diff
@@ -4,7 +4,7 @@
 import logging
 import os
 import sgmllib
-import urllib2
+import requests
```

Comment:
There is a requests controller - so this isn't getting the requests top-level package. I'll open a PR with a fix.

Reply:
Oh jeez, ok, good catch. Thanks John